src/index.tsx

```diff
@@ -214,8 +214,14 @@ function App() {
     [isOnMapHoverEventEnabled, justClicked],
   );
 
+  let height = String(mapHeight);
+  if (typeof mapHeight === "number") {
+    // if a number is used for height, assume pixels
+    height = height + "px";
+  }
```

Comment:
I was thinking of suggesting this validation be done on the Python side, so the Python side is always sending a string to the JS side. There are several places that we already use custom validators